examples/integration/src/scene.rs

```diff
@@ -79,6 +78,7 @@ fn build_pipeline(device: &wgpu::Device) -> wgpu::RenderPipeline {
                 depth_bias: 0,
```

Comment:
may omit the depth stuff since it's default as well

Reply:
Done in b689778ed94b34706bf97e5994a721a8648386a5.